src/compiler/compile/nodes/Element.ts

```diff
@@ -184,7 +184,7 @@ export default class Element extends Node {
 
 				case 'Attribute':
 				case 'Spread':
-					// special case
+				// special case
```

Comment:
oh interesting, it's indented the code and not the comments. Is there an eslint rule to configure this? I don't use comments.